en/snap/platform/overview.md

```diff
@@ -333,6 +333,19 @@ Few points to understand about order status & item stock management that is mana
 - It is recommended to **avoid manual order status changes (manual intervention)** from Shopify Admin Panel at least between period of the order first created as `pending` and it finally become `paid/canceled` (about 0-26 hours), in order for payment integration with Midtrans to perform smoothly. Which the order status and item stock will be managed automatically based on the flow explained on this page.
   - Manual order status changes may cause unexpected behaviour in terms of order status & item stock management, such as order status stuck at certain state. Do this at your own risk. Midtrans may not be in position to help/explain with the consequences.
 
+##### Email Notification SNAP Validation Error
+When a customer selects credit card on the snap payment page, then the payment is denied/rejected by the bank, then the customer decided to left without trying to select another payment method/use another credit card, you as a merchant maybe will receive an email notification with the subject __"Snap - Validation Error"__ with the message: __"Transaction can't be canceled after being paid"__. 
+
+For now, it's safe to ignore the email, because Midtrans will retry to invalidate the snap payment page after two hours if without any successful payment.
+
+##### Customer still able to retry payment if status order is expired
+An order with status expired, customer still able to retry payment if the order receive failure notification payment from Midtrans. If in the timeline history of the order, you find the notification e.g: `Unable to process a payment for Rp220,000.00 IDR on Midtrans Payment`, your customer will receives an email to let them know that their payment couldn't be processed. The email includes a Pay now link that the customer can use to try paying again.
+
+##### One order have 2 payment ID
+Yes, it's possible. it same case with [Customer still able to retry payment if status order is expired](/en/snap/platform/overview?id=customer-still-able-to-retry-payment-if-status-order-is-expired).
```

Comment:
- There is not even a question. Why are you startin sentence with `Yes,`
- You are not properly using the correct link format. Read Readme.